### ISSUE TITLE
Getting the schema of the "datadog" plugin does not produce a JSON response

### ISSUE THREAD

**Issue by lmika** [Kong/kong#3949]
### Summary

Attempting to get the schema of the "datadog" plugin when calling the "/plugins/schema/datadog" endpoint on the admin API directly returns a 200 OK response but without a JSON body:

```
$ curl -v http://localhost:8001/plugins/schema/datadog
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8001 (#0)
> GET /plugins/schema/datadog HTTP/1.1
> Host: localhost:8001
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Mon, 05 Nov 2018 01:19:07 GMT
< Content-Type: application/json
< Connection: keep-alive
< Access-Control-Allow-Origin: *
< Server: kong/1.0.0rc2
< Content-Length: 0
< 
* Connection #0 to host localhost left intact
```

The following message is produced within the Kong error logs:

```
2018/11/05 01:19:07 [debug] 4887#0: *32 [lua] init.lua:307: No API endpoints loaded for plugin: statsd
2018/11/05 01:19:07 [debug] 4887#0: *32 [lua] init.lua:299: Loading API endpoints for plugin: prometheus
2018/11/05 01:19:07 [debug] 4887#0: *32 [lua] init.lua:307: No API endpoints loaded for plugin: request-termination
2018/11/05 01:19:07 [error] 4887#0: *32 [lua] responses.lua:160: [admin] could not encode value: Cannot serialise number: must not be NaN or Infinity, client: 10.0.2.2, server: kong_admin, request: "GET /plugins/schema/datadog HTTP/1.1", host: "localhost:8101"
2018/11/05 01:19:07 [info] 4887#0: *32 client 10.0.2.2 closed keepalive connection
```

This happens on a vanilla instance of Kong 1.0.0rc2, running with PostgreSQL on Linux with only the bundled plugins preloaded.

### Steps To Reproduce

1. Install Kong Community Edition ver. 1.0.0rc2 from the [CentOS 6 RPM](https://bintray.com/kong/kong-community-edition-rpm/download_file?file_path=centos%2F6%2Fkong-community-edition-1.0.0rc2.el6.noarch.rpm)
2. Set `plugins = bundled` in Kong configuration
3. Initialise DB and start Kong
4. Run `curl http://localhost:8001/plugins/schema/datadog`

### Additional Details & Logs

- Kong version = `1.0.0rc2`
- Kong debug-level startup logs = [kong-start.log attached](https://github.com/Kong/kong/files/2546571/kong-start.log)
- Kong error logs = [error.log attached](https://github.com/Kong/kong/files/2546573/error.log)
- Kong configuration = [config.txt attached](https://github.com/Kong/kong/files/2546576/config.txt)
- Operating system = `Amazon Linux AMI release 2017.03`

**Comment by julienlau** [comment]
Any idea on how to manually solve this ? 
Is this linked to some value stored into postsgres ?

**Comment by hishamhm** [comment]
Fixed in #4162 — thanks for reporting!
